### PR TITLE
Update Tomcat Install scripts

### PR DESCRIPTION
**Pull request by Mrod1598**
The current Tomcat integration tests are flawed in how the tars are pulled. the mirror deletes the previous version when a new one is uploaded. So, changed to using packages instead. Also, added sleep statement at beginning of exercise script to help with flakiness of not being able to hit the endpoint sometimes.

### CODE REVIEW

File: apps/tomcat.go
Comment (qingling128) [inline apps/tomcat.go:168]: What impact does this change have? Looks like we are changing the default log file path. Does any distro use `/opt/tomcat/logs/catalina.out` instead? If so, should we include both?

**Comment by ridwanmsharif** [comment]
The sleep part of this PR will reduce the flakiness we're seeing in the third party integration tests. This is currently blocking our 2.9.0 release of the Ops Agent

File: apps/tomcat.go
Comment (Mrod1598) [reply apps/tomcat.go:168]: I'll Add both, it seems the package manager uses what I changed it to and I'm seeing more and more the `/var/log` but I've seen both and tomcat doesn't explicitly call it out in the docs that I can tell so I'll add both.

Comment (quentinmit) [reply apps/tomcat.go:168]: What about tomcat10? Or tomcat8? I think we're going to need multiple file paths to cover different versions and distros.

File: integration_test/third_party_apps_data/applications/tomcat/centos_rhel/install
Comment (quentinmit) [inline integration_test/third_party_apps_data/applications/tomcat/centos_rhel/install:5]: Does the tomcat package not come with a systemd unit? Either way you shouldn't be overwriting anything in `/usr/lib`. If you need to add something to the unit file, you should be writing an override file in `/etc/systemd/system` (and there should be a good reason to do that).

**Review by quentinmit — CHANGES_REQUESTED** [review]

**Comment by Mrod1598** [comment]
Clossing This PR because https://github.com/GoogleCloudPlatform/ops-agent/pull/342 seems to fix it. I'll address the Include path issues in a separate PR.

**Comment by quentinmit** [comment]
I think this PR is preferable to #342

**Comment by Mrod1598** [comment]
> I think this PR is preferable to #342

yes, however, if this is blocking a release I'd rather have the other one merged while I investigate a better way to handle this as it is still failing on this branch.

**Comment by qingling128** [comment]
+1 that #344 is preferable to #342. I'd prefer the user visible changes (aka the `includePaths` changes`) to go in this release as well. The installation related changes are fine to get in after the release.

File: integration_test/third_party_apps_data/applications/tomcat/centos_rhel/install
Comment (Mrod1598) [reply integration_test/third_party_apps_data/applications/tomcat/centos_rhel/install]: Updated and there needs to be updated to the service file to enable jmx.

**Comment by Mrod1598** [comment]
> +1 that #344 is preferable to #342. I'd prefer the user visible changes (aka the `includePaths` changes`) to go in this release as well. The installation related changes are fine to get in after the release.

reopened and updated log paths.

File: docs/tomcat.md
Comment (qingling128) [inline docs/tomcat.md:60]: We changed this to wildcard in #342, right?

Comment (Mrod1598) [reply docs/tomcat.md:60]: yes, looks like the doc update was missed

**Comment by qingling128** [comment]
The Tomcat tests seem to be failing: 
image

File: integration_test/third_party_apps_data/applications/tomcat/centos_rhel/install
Comment (quentinmit) [inline integration_test/third_party_apps_data/applications/tomcat/centos_rhel/install:5]: At least this is in /etc instead of /usr, but please use a systemd drop-in and just customize the one thing you need (the environment variables?). See https://www.freedesktop.org/software/systemd/man/systemd.unit.html#id-1.14.3

**Review by quentinmit — DISMISSED** [review]

File: integration_test/third_party_apps_data/applications/tomcat/centos_rhel/install
Comment (qingling128) [reply integration_test/third_party_apps_data/applications/tomcat/centos_rhel/install]: Thanks!

**Review by qingling128 — APPROVED** [review]
LGTM modulo the integration tests pass

**Comment by qingling128** [comment]
Gentle ping on this PR. We are trying to do a release today, and it's blocked by the Tomcat failures.

**Comment by Mrod1598** [comment]
CentOS script has to be left as is as the only available package is tomcat7 which we don't support

File: integration_test/third_party_apps_data/applications/tomcat/centos_rhel/install
Comment (qingling128) [reply integration_test/third_party_apps_data/applications/tomcat/centos_rhel/install]: This one seems obsolete now. It later turned out that the only available package is tomcat7 on CentOS, which we don't support. So this change has been reverted.

**Pull request merged**
